web-app/css/source/assessment.css

```diff
@@ -13,14 +13,13 @@
     font-size: 1em;}
 .form-labels { display: table-cell; width: 150px;    font-family: helvetica;
     font-size: 1em;}
-.form-inputs { display: table-cell; width: 200px;     font-family: helvetica;
+.form-inputs { display: table-cell; width: 400px;     font-family: helvetica;
```

Comment:
rules should be on a new line after selector, and each rule should have its own line, rules should be tabbed within selector context.
``` css
bad
.example { display: none; color: blue; }

good
.example {
   display: none;
   color: blue;
}
```